lib/httpoison_adapter.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GlassFactoryApi.HttpoisonAdapter do
```

Comment:
🎨 (as to be consistent with the lib name)

```suggestion
defmodule GlassFactoryApi.HTTPoisonAdapter do
```